Woof/Woof/NetworkLayer/Endpoint/WoofAppEndpoint+HTTPAPIEndpoint.swift

```diff
@@ -15,19 +15,22 @@ extension WoofAppEndpoint: HTTPAPIEndpoint {
 
     var path: String {
         switch self {
-        default: return ""
+        case .addNewSitter: return EndpointPath.addNewSitter
         }
     }
 
     var task: HTTPTask {
         switch self {
-        default: return .request
+        case let .addNewSitter(sitter):
+            // transform sitter to body parameters
+            let bodyParameters = try? sitter.asDictionary()
+            return .requestWithBodyParameters(bodyParameters: bodyParameters)
```

Comment:
Maybe it's better to use `BodyParameters` as associated value of Woof endpoint's cases and previously to transform a `Sitter` object to check the success of the transformation. It lets not to send the request, if something went wrong.
WDYT?
```suggestion
        case let .addNewSitter(bodyParameters):
            return .requestWithBodyParameters(bodyParameters: bodyParameters)
```

Reply:
We need to talk about this on call.

Reply:
It seems to me that your current solution is pretty good.

Reply:
As discussed, the associated value for `addNewSitter(parameters)` case was changed.
Move the extension for Encodable with `asDictionary` method to `Helper` folder, as it is not related to `WoofAppEndpoint` now.